release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -2730,6 +2730,49 @@ This section will continue to be updated over time to provide notes on enhanceme
 ====
 
+[id="ocp-4-15-17"]
+=== RHBA-2024:3673 - {product-title} 4.15.17 bug fix update and security update
+
+Issued: 2024-06-11
+
+{product-title} release 4.15.17 is now available. The list of bug fixes that are included in this update is documented in the link:https://access.redhat.com/errata/RHBA-2024:3673[RHBA-2024:3488] advisory. The RPM packages that are included in this update are provided by the link:https://access.redhat.com/errata/RHSA-2024:3676[RHSA-2024:3676] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.15.17 --pullspecs
+----
+
+[id="ocp-4-15-17-enhancements"]
+==== Enhancements
+
+The following enhancement is included in this z-stream release:
+
+[id="ocp-4-15-17-kube-api-server-v1alpha1-removal-prep"]
+===== Storage migration for cluster versions 4.8 or earlier
+
+* This release introduces a storage migration that supports a safe cluster update from version 4.8 or earlier to the latest supported release. If you created a cluster at version 4.7 or earlier, your stored objects are accessible when updating your cluster to the latest supported release. (link:https://issues.redhat.com/browse/OCPBUGS-31445[*OCPBUGS-31445*])
+
+[id="ocp-4-15-17-bug-fixes"]
+==== Bug fixes
+
+* Previously, when multiple domains were configured for an Amazon Virtual Private Cloud (VPC) DHCP option, the hostname could return multiple values. The logic did not account for multiple values, and crashed when it returned a node name containing a space. with this release, the logic has been updated to use only the first returned hostname as the node name and the issue has been resolved. (link:https://issues.redhat.com/browse/OCPBUGS-33847[*OCPBUGS-33847*])
+
+* Previously, when enabling `virtualHostedStyle` with `regionEndpoint` set in the image registry operator configuration, the image registry ignored the virtual hosted style configuration and failed to start. With this release, the image resistry uses a new upstream distribution configuration and the issue has been resolved. (link:https://issues.redhat.com/browse/OCPBUGS-34539[*OCPBUGS-34539*])
+
+* Previously, OperatorHub incorrectly excluded the Amazon Resource Name (ARN) role information for ROSA Hosted Control Plane (HCP) clusters. With this update, OperatorHub correctly displays ARN information and the issue has been resolved. (link:https://issues.redhat.com/browse/OCPBUGS-34550[*OCPBUGS-34550*])
+
+* Previously, when attempting to delete a cluster or BareMetalHost (BMH) before installation, the metal3-operator tried to unnecessarily generate a pre-provisioning image. With this release, an exception has been created to prevent the creation of a pre-provisioning image during a BMH deletion and the issue has been resolved. (link:https://issues.redhat.com/browse/OCPBUGS-34682[*OCPBUGS-34682*])
```

Comment:
```suggestion
* Previously, when attempting to delete a cluster or BareMetalHost (BMH) resource before installation, the metal3-operator tried to unnecessarily generate a pre-provisioning image. With this release, an exception has been created to prevent the creation of a pre-provisioning image during a BMH deletion and the issue has been resolved. (link:https://issues.redhat.com/browse/OCPBUGS-34682[*OCPBUGS-34682*])
```